src/commands/setup-web.js

```diff
@@ -9,10 +9,19 @@ var requirePermissions = require("../requirePermissions");
 
 var JS_TEMPLATE = fs.readFileSync(__dirname + "/../../templates/setup/web.js", "utf8");
 
+/**
+ * This command is deprecated in favor of `apps:sdkconfig web` command
+ * TODO: Remove this command after 02/09/2020
```

Comment:
Would you remove the date from this TODO?

Reply:
Done